UnityProject/Assets/Scripts/Electricity/PoweredDevices/APC.cs

```diff
@@ -53,9 +53,18 @@ private void SetVoltage(float newVoltage)
 		UpdateDisplay();
 	}
 
+	 private void OnStartServer()
+	{
+		SetVoltage(this.voltageSync);
+
+		//if extending another component
+		base.OnStartServer();
+	}
```

Comment:
remember that OnStartClient() is run on the server

Reply:
It should be fine , Since it's Setting to the same value?